content/12-target-details.md

```diff
@@ -1945,6 +1945,56 @@ java -jar bin/Main.jar
 
 
 
+<!--label:target-jvm-->
+### JVM
+
+<!--subtoc-->
+
+<!--label:target-jvm-getting-started-->
+#### Getting started with Haxe/JVM
+
+To get started with Haxe/JVM, create a new folder and save this class as `Main.hx`.
+
+[code asset](assets/HelloWorld.hx)
+
+To compile Haxe to JVM bytecode, you'll first need to install the [hxjava haxelib](http://lib.haxe.org/p/hxjava). This is the Haxe Java support library, including build scripts and support code. Then run the following from the command line:
+
+```sh
+haxe --jvm bin/Main.jar --main Main
+```
+
+To save yourself some typing, you may create a `.hxml` file (say, `jvm.hxml`) in your project's top-level directory, containing:
+
+```hxml
+--jvm bin/Main.jar
+--main Main
+```
+
+and run it with the `haxe` command:
+
+```sh
+haxe jvm.hxml
+```
+
+to produce the bin/Main.jar file.
+
+To execute the jar file, you'll need to have a JRE (Java Runtime Environment) installed, such as [OpenJDK](https://openjdk.java.net/) or a commercially-supported JRE. Then run the jar file using the following command:
+
+```sh
+java -jar bin/Main.jar
+```
+
+Note that some operating systems may allow you to double-click a jar file to run it.
```

Comment:
That information is pretty random, I don't think we need that here.

Reply:
I Agree.

Reply:
Removed.